<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db.sqlite3" uuid="ce0f1cd5-d121-4d68-af91-6a231ad87071">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="9946bb2a-31a9-4716-9ec8-dcda2ea05370">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/FootAi/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\Administrator\PycharmProjects\FootAi\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
  </component>
</project>